backend/pkg/models/rentACar.go

```diff
@@ -31,8 +30,8 @@ type FindVehicleParams struct {
 	Type      string
 	PriceLow  float64
 	PriceHigh float64
-	StartDate time.Time
-	EndDate   time.Time
+	StartDate string
```

Comment:
Zašto je ovo menjano?
Datumska reprezentacija je preciznija.

Reply:
Zato što ovaj čovek ovde ne ume drugačije da implementira